lib/Utils/Log.php

```diff
@@ -30,6 +74,18 @@ public static function consoleDebug($data, $postpone = true, $title = 'PHP', $lo
         }
     }
 
+    /**
+    * Pretty prints a variable.
+    *
+    * Prints human-readable information about a variable.
+    * Print will be wrapped between <pre> tags.
+    * Will also output the data type, the filename and line number.
+    *
+    * @since 0.2.0
```

Comment:
what do we do with something like this? `0.2.0` is not even released yet, so this number could be anything